selenium_tests/course_info/course_info_remove_people_tests.py

```diff
@@ -43,5 +43,15 @@ def test_remove_person(self, test_case_id, test_user, canvas_role, role_id):
         self.people_page.delete_user(test_user)
 
         self.assertTrue(self.people_page.delete_was_successful())
+
+        '''Note: Page needs to reloaded as tests are persistently failing
+        for element not found - perhaps cache has changed" for different test
+        id (as the test ID are being removed so DOM has changed). This ensures
+        that the page is reloaded.'''
+        self.driver.refresh()
+        self.setUp()
+        self._load_test_course()
```

Comment:
I think that to reload the page you don't need the `refresh()`, right? 

In general, I agree that this seems overkill to deal with the stale element exception. I also am not sure where that stale element exception comes from, it would be worth further investigation -- but if this fixes it, let's get it running and worry about efficiency later.